lib/charms/opensearch/v0/opensearch_health.py

```diff
@@ -58,42 +61,87 @@ def apply(
         app: bool = True,
```

Comment:
@Mehdi-Bendriss we should default this to False, as `self._apply_for_app()` changes the peer relation directly

Reply:
Same answer as https://github.com/canonical/opensearch-operator/pull/279#discussion_r1589336622